### PR TITLE
React & Redux group project

### PR DESCRIPTION
**Pull request by yigitm**
We were two teammates in this project @jonathastavares  & @yigitm.

- All kanban board tasks are completed
- Unit & component tests are added

Thank you for your time.

### CODE REVIEW

File: src/components/Nav.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+          </NavLink>
+        </li>
+      </ul>
+    </nav>
+    <Routes>
+      <Route path="/" element={<Rockets />} />
+      <Route path="/missions" element={<Missions />} />
+      <Route path="/profile" element={<Profile />} />
+    </Routes>
+  </Router>
```
Comment (omar-labana) [inline src/components/Nav.js:60]: - [x] _[OPTIONAL]_ Kindly, separate the routing config from the navigation, I'd suggest moving the routing to the `App` component and making `Nav` its own standalone component.

File: src/components/Rockets.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      </Col>
+    </Row>
+  ));
+
+  return (
+    <Container fluid className="mt-5">
+      {displayRockets}
+    </Container>
+  );
+};
```
Comment (omar-labana) [inline src/components/Rockets.js]: - [x] Kindly, make sure to render rockets conditionally using React conditional rendering syntax
`{rockets  && ( 
    // rockets 
)}
`
- [x] _[OPTIONAL]_ I'd to you divide this component into smaller pieces, I feel it has too much logic going on.

File: src/components/missions/Missions.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+            <th className="fs-3 border" scope="col"> </th>
+          </tr>
+        </thead>
+        <tbody>
+          {missions.map((mission) => (
+            <Mission key={mission.mission_id} mission={mission} />
+          ))}
+        </tbody>
+      </table>
+    </div>
```
Comment (omar-labana) [inline src/components/missions/Missions.js:25]: - [x] Kindly, make sure to render missions conditionally using React conditional rendering syntax
`{missions  && ( 
    // missions 
)}
`

**Review by omar-labana — CHANGES_REQUESTED** [review]
Hi team, 

Excellent work so far! 👌
You have implemented all the required features demonstrating PERFECT knowledge of Redux. ✔️ 
The git-flow is nicely done and README is quite descriptive. 👌
There are some cases that you need to adjust but you are almost done! 😀

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Rockets.js (hunk 2 above)
Comment (yigitm) [reply src/components/Rockets.js]: Thank you for the review. I will implement the optional changes in the near future.

File: src/components/Nav.js (hunk 1 above)
Comment (yigitm) [reply src/components/Nav.js:60]: Thank you for the review. I will implement the optional changes in the near future.

**Review by Emmyn5600 — APPROVED** [review]
Hi @yigitm and @jonathastavares 

Woo ow 👏, Your project is complete!  There is nothing else to say other than... it's time to merge it. 👏

Congratulations! 🎉 

##  Optional suggestion

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**